server/orders/models.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import unicode_literals
+from django.db import models
+from users.models import User
+from chefs.models import Chef
+from menus.models import Menu, Item
+from billing.models import Address
+from django.db.models import Sum
+
+class Order(models.Model):
+    '''
+    Represents a created order.
+    '''
+    customer = models.ForeignKey(User, help_text='Foreign key to the user who'
```

Comment:
do you think we should rename this field to user instead of customer just to keep it consistent?